habitat_baselines/common/obs_transformers.py

```diff
@@ -84,6 +84,11 @@ def __init__(
         self._size: int = size
         self.channels_last: bool = channels_last
         self.trans_keys: Tuple[str, ...] = trans_keys
+        self.interpolation_mode = {
+            "rgb": "area",
+            "depth": "area",
+            "semantic": "nearest-exact",
```

Comment:
You may want to encode a default of area.

Reply:
@Skylion007 but is it right to use the area for semantic sensor observations? It might create non-int values at some places which might not work if you are trying to pass semantic observations through a embedding layer in your policy

Reply:
I don't like using a dict here because it's not robust to multiple RGB sensors or depth cameras etc which is a common use case.

Reply:
@vincentpierre @Skylion007 if we want to move interpolation mode to config where do you recommend we put it? I am not sure how we can move the interpolation mode to `ENABLED_TRANSFORMS.RESIZE_SHORTEST_EDGE`, this config is specific to a sensor so we would need to put it for each sensor in the transforms config.

Reply:
Yeah, if I redesigned the OBS_TRANSFORMER I would have attached the keys to the sensors. You can specify the keys for individual transforms, but not individual args. @vincentpierre This probably would be a good aspect to redesign as it's a recurring problem.

@Ram81 We ran into this issue with Equirect ObsTransformers and used DTYPE as hack there to detect if it's was a semantic sensor I think

Reply:
I thought there was a clean way to do this. If there is none, maybe what you have is the best solution

Reply:
The "clean" way of doing it previously was detecting dtype, but we really should have a cleaner way to do it.

Reply:
@vincentpierre @Skylion007 yes, currently. This looked like the only way to do this. I can look into how `Equirect ObsTransformers` handles this and see if I can make it any cleaner than this.